content/en/docs/collector/getting-started.md

```diff
@@ -1,358 +1,90 @@
-tar -xvf otelcol_{{% param collectorVersion %}}_linux_386.tar.gz
-```
-
-{{% /tab %}} {{% tab ppc64le %}}
-
-```sh
-curl --proto '=https' --tlsv1.2 -fOL https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_linux_ppc64le.tar.gz
-tar -xvf otelcol_{{% param collectorVersion %}}_linux_ppc64le.tar.gz
-```
-
-{{% /tab %}} {{< /tabpane >}}
-
-### Automatic Service Configuration
-
-By default, the `otelcol` systemd service will be started with the
-`--config=/etc/otelcol/config.yaml` option after installation. To customize
-these options, modify the `OTELCOL_OPTIONS` variable in the
-`/etc/otelcol/otelcol.conf` systemd environment file with the appropriate
-command-line options (run `/usr/bin/otelcol --help` to see all available
-options). Additional environment variables can also be passed to the `otelcol`
-service by adding them to this file.
-
-If either the Collector configuration file or `/etc/otelcol/otelcol.conf` are
-modified, restart the `otelcol` service to apply the changes by running:
-
-```sh
-sudo systemctl restart otelcol
-```
-
-To check the output from the `otelcol` service, run:
-
-```sh
-sudo journalctl -u otelcol
-```
-
-## MacOS Packaging
-
-MacOS [releases][] are available for Intel- & ARM-based systems. They are
-packaged as gzipped tarballs (`.tar.gz`) and will need to be unpacked with a
-tool that supports this compression format:
-
-{{< tabpane text=true >}} {{% tab Intel %}}
-
-```sh
-curl --proto '=https' --tlsv1.2 -fOL https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_darwin_amd64.tar.gz
-tar -xvf otelcol_{{% param collectorVersion %}}_darwin_amd64.tar.gz
-```
-
-{{% /tab %}} {{% tab ARM %}}
-
-```sh
-curl --proto '=https' --tlsv1.2 -fOL https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v{{% param collectorVersion %}}/otelcol_{{% param collectorVersion %}}_darwin_arm64.tar.gz
-tar -xvf otelcol_{{% param collectorVersion %}}_darwin_arm64.tar.gz
+git clone git@github.com:open-telemetry/opentelemetry-collector-contrib.git --depth 1; \
+  cd opentelemetry-collector-contrib/examples/demo; \
+  docker compose up -d
 ```
 
-{{% /tab %}} {{< /tabpane >}}
-
-Every Collector release includes an `otelcol` executable that you can run after
-unpacking.
-
-## Windows Packaging
-
-Windows [releases][] are packaged as gzipped tarballs (`.tar.gz`) and will need
-to be unpacked with a tool that supports this compression format.
-
-Every Collector release includes an `otelcol.exe` executable that you can run
-after unpacking.
-
-## Local
+You can find more information about the demo in the [README.md][] file.
```

Comment:
Let's remove that section about the demo

Reply:
Done in https://github.com/open-telemetry/opentelemetry.io/pull/3412/commits/c1b457308a14b07867b4f414b2ba5ff67cdbb541